app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -124,7 +146,7 @@ def create_security_group_queue(userid, options = {})
     }
     queue_opts = {
       :class_name  => self.class.name,
-      :method_name => 'create_security_group',
+      :method_name => 'create_floating_ip',
```

Comment:
Yep - I'd fix it here, except there's a separate upstream PR that fixes this issue: https://github.com/ManageIQ/manageiq/pull/12653, so I figured it's better to just leave it to that.